src/state/presentationManagerImpl.ts

```diff
@@ -54,6 +54,9 @@ export class PresentationManagerImpl implements PresentationManager {
 
   private activeSlidePublisher: Subscription | undefined;
 
+  private isEditMode: boolean = false;
+  private isEditModeSubject = new BehaviorSubject<boolean>(false);
```

Comment:
No need to explicitly specify the type in this case.

```suggestion
  private isEditModeSubject = new BehaviorSubject(false);
```